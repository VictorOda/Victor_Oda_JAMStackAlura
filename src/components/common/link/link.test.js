import React from 'react';
import { screen } from '@testing-library/react';
import Link from './link';
import { render } from '../../../infra/tests/testUtils';

describe('<FormContact />', () => {
  test('renderiza componente', () => {
    render(
      <Link href="/projects/quiz-challenge">
        test
      </Link>,
    );
    screen.debug();
    const link = screen.getByRole('link', {
      name: /test/i,
    });
    expect(link).toMatchSnapshot();
  });
});
