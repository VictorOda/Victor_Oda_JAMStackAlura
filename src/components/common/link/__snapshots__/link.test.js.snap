// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormContact /> renderiza componente 1`] = `
.c0 {
  color: inherit;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  opacity: 1;
  -webkit-transition: opacity 200ms ease-in-out;
  transition: opacity 200ms ease-in-out;
  cursor: pointer;
}

.c0:hover,
.c0:focus {
  opacity: .5;
}

<a
  class="c0"
  href="/projects/quiz-challenge"
>
  test
</a>
`;
